pullrequest.js

```diff
@@ -55,6 +62,7 @@ function getId(path) {
 }
 
 function toggleDiff(id, duration, display) {
+    var prId = getCurrentPrId();
```

Comment:
https://github.com/Yatser/prettypullrequests/pull/36/files#diff-658ced428d8d6f3cb15c731677e9f305R160

Putting it right above `injectHtml()` is probably good.